### ISSUE TITLE
Pass options to the custom importer

### ISSUE THREAD

**Issue by strarsis** [sass-eyeglass/eyeglass#83]
As separate issue from  https://github.com/sass-eyeglass/eyeglass/issues/42#issuecomment-156851307

I try to use this custom importer with eyeglass: https://github.com/at-import/node-sass-import-once

````
[...]
  var eyeglass = new Eyeglass({
    importer: importOnce,
    importOnce: {
      index: true,
      css:   false,
      bower: false
    }
  });
[...]
````

However, the importer isn't able to see the importOnce property in options object, 
is there a way to pass it to the custom importer when eyeglass is used instead of plain node-sass?

````
/home/build/src/test/node_modules/node-sass-import-once/index.js:275
  if (!this.options.importOnce) {
                   ^

TypeError: Cannot read property 'importOnce' of undefined
    at Object.importer [as fallbackImporter] (/home/build/src/test/node_modules/node-sass-import-once/index.js:275:20)
[...]
````

Thank you and
With best regards

**Comment by eoneill** [comment]
Thanks for reporting. The issue is that we manually invoke the importers, but we don't bind it to the context (`this`) that node-sass is invoking them with. I have a test case and fix I'll PR once #72 is merged.

**Comment by strarsis** [comment]
See the referenced issue for a workaround that involves a wrapper that is bound.

With best regards
